File: Programming-Basics-with-Python/2.First-Steps-in-Coding-Exercise/03.DepositCalculator.py
# входни данни -> депозирана сума (float), срок (int), процент (float)
# изчисляваме крайна сума
# отпечатваме крайната сума
deposit_sum = float(input())
months = int(input())
percent = float(input())

# сума = депозирана сума  + срок на депозита * ((депозирана сума * годишен лихвен процент ) / 12)
final_sum = deposit_sum + months * ((deposit_sum * (percent / 100)) / 12)
print(final_sum)

File: Programming-Basics-with-JavaScript/1.First-Steps-in-Coding/03depositCalculator.js
function calculator(input){
    let depositSum = Number(input[0])
    let months = Number(input[1])
    let percent = Number(input[2])
    let sum = depositSum + months *((depositSum * percent / 100) / 12)
    console.log(sum)
}
calculator(["200 ", "3 ", "5.7 "])